sdk-python/littlehorse/workflow.py

```diff
@@ -1871,56 +1969,29 @@ def do_if(
         self._check_if_active()
         self._validate_initializer(if_body)
 
-        # execute if branch
-        start_node_name = self.add_node("nop", NopNode())
-        self._last_node_condition = condition.compile()
-        if_body(self)
-
-        last_condition_from_if_block = self._last_node_condition
-        last_node_from_if_block = self._last_node()
-        variables_from_if_block = self._collect_variable_mutations()
-
-        start_node = self._find_node(start_node_name)
+        # Adds new chain-able else if functionality
+        if else_body is None:
+            return self._do_if(condition, if_body)
 
-        # execute else branch
-        if else_body is not None:
-            self._validate_initializer(else_body)
+        self._validate_initializer(else_body)
+        self._do_if(condition, if_body).do_else(else_body)
+        return None
```

Comment:
Are you sure about this? this could break the customer code, shouldn't return WorkflowIfStatement always?